### PR TITLE
Tutorial with modified plugin

### PR DESCRIPTION
**Pull request by wighawag**
**Description**
Use a modified version of the plugin, see : https://github.com/ethereum-optimism/plugins/pull/35

Making it overall simpler to configure

### CODE REVIEW

**Comment by platocrat** [comment]
This looks great and much simpler than what we had.
Thanks a ton for this.
I'll wait on @smartcontracts to review and merge [ethereum-optimism/plugins#35](https://github.com/ethereum-optimism/plugins/pull/35) before merging this

File: README.md
Comment (tynes) [inline README.md:105]: Typo here `hae` -> `have`

**Review by platocrat — APPROVED** [review]

**Pull request merged**
